test/torchaudio_unittest/prototype/hdemucs_test_impl.py

```diff
@@ -106,3 +116,46 @@ def test_decoder_output_shape_time(self):
 
         assert z.size() == (batch_size, chout, t * stride)
         assert y.size() == (batch_size, chin, t)
+
+
+@skipIfNoModule("demucs")
+class CompareHDemucsOriginal(TorchaudioTestCase):
+    """Test the process of importing the models from demucs.
+
+    Test methods in this test suite will check to assure correctness in factory functions,
+    comparing with original hybrid demucs
+    """
+
+    def _get_original_model(self, sources: List[str], nfft: int, depth: int):
+        from demucs import hdemucs as original
+
+        original = original.HDemucs(sources, nfft=nfft, depth=depth)
+        return original
+
+    def _assert_equal_models(self, factory_hdemucs, depth, nfft, sample_rate, sources):
+        torch.random.manual_seed(0)
+        original_hdemucs = self._get_original_model(sources, nfft, depth).to(self.device).eval()
+        inputs = _get_inputs(sample_rate=sample_rate, device=self.device)
+        factory_output = factory_hdemucs(inputs)
+        original_output = original_hdemucs(inputs)
+        self.assertEqual(original_output, factory_output)
+
+    @SOURCES_OUTPUT_CONFIG
+    def test_import_recreate_low_model_test(self, sources):
```

Comment:
nit (same with for high model), two tests in function name is a little redundant

```suggestion
    def test_import_recreate_low_model(self, sources):
```